cmd/ethereum_exporter/main.go

```diff
@@ -49,7 +50,12 @@ func main() {
 		os.Exit(0)
 	}
 
-	erc20Address := common.HexToAddress(*erc20ContractAddress)
+	var erc20Addresses []common.Address
+	stringAddresses := strings.Split(*erc20ContractAddresses, ",")
+	for _, stringAddr := range stringAddresses {
```

Comment:
Maybe validate entries?

Reply:
That a valid address is generated?

Reply:
Exactly, we're blindly accepting user input as valid addresses.

```go
if common.IsHexAddress(stringAddr) { ... }
```

Docs [here](https://pkg.go.dev/github.com/ethereum/go-ethereum/common#IsHexAddress).

Reply:
Good one